spec/apps/rails4.rb

```diff
@@ -0,0 +1,48 @@
+require "rails"
+require 'rails/all'
+require 'action_view/testing/resolvers'
+require 'rails/test_help'
+
+require 'ror_push_notifications' # our gem
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.